deutsche-bank-adapter/src/main/java/de/adorsys/xs2a/gateway/service/impl/JsonMapper.java

```diff
@@ -0,0 +1,40 @@
+package de.adorsys.xs2a.gateway.service.impl;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+
+public class JsonMapper {
```

Comment:
I propose to discuss this solution with @spetrychenko , as his PR #16 has conflicts with current PR.